### PR TITLE
Add languages guide

### PR DESCRIPTION
**Pull request by strarsis**
Add a languages guide page.

(@swalkinshaw, @retlehs, @odinho: Previous PR https://github.com/roots/docs/pull/191)

### CODE REVIEW

File: docs/trellis/master/languages.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
```
Comment (swalkinshaw) [inline docs/trellis/master/languages.md]: Think there's a space here or something breaking the rendering. See at https://github.com/roots/docs/blob/3a01d149dabd6cb590f25012eb5d6e87fa3a9e7b/docs/trellis/master/languages.md

And on the preview https://deploy-preview-307--hardcore-pike-9e5241.netlify.app/docs/trellis/master/languages/

Comment (strarsis) [reply docs/trellis/master/languages.md]: Yes, it seems some typo got in there. It should be fixed now.

**Comment by swalkinshaw** [comment]
🎉  thank you for this

**Comment by strarsis** [comment]
The new docs would be probably released later, right? Because the current docs don't contain a "Guides" section or that page yet:
https://roots.io/docs/trellis/master/installation/

**Comment by swalkinshaw** [comment]
Hmm I figured it was a service worker caching issue but that doesn't seem to be the case. Maybe there's something different with a new page/section

@Log1x do you know?

**Comment by retlehs** [comment]
thanks for doing this! fyi i made some [minor changes](https://github.com/roots/docs/commit/c3f8ac3c6e5c0a165c970575702a2701a5825289) and moved it here:

https://docs.roots.io/trellis/master/guides/install-wordpress-language-files/

**Pull request merged**